tck/src/main/resources/tests/favouriteDrinkSizeML/input.graphql

```diff
@@ -0,0 +1,6 @@
+mutation updateFavouriteDrinkSizeForHero {
```

Comment:
Needs cleanup, but good that it is different from the other favouriteDrinkSize test.